src/jdk.jdwp.agent/share/native/libjdwp/debugInit.c

```diff
@@ -877,7 +877,8 @@ printUsage(void)
  "onthrow=<exception name>         debug on throw                    none\n"
  "onuncaught=y|n                   debug on any uncaught?            n\n"
  "timeout=<timeout value>          for listen/attach in milliseconds n\n"
- "enumeratevthreads=y|n            thread lists include all vthreads n\n"
+ "enumeratevthreads=y|n            (Preview Feature) thread lists include all vthreads\n"
```

Comment:
I think the description needs to say "virtual threads" rather than "vthreads". Also I think "(Preview" needs to go after "virtual threads" to make it clearer that it's virtual threads that are preview,.

Reply:
I was really trying to emphasize that the flag is preview, and therefore may change or go away.  If I change as  you suggest, the reader could imply that as long as virtual threads support is in place, then the flag will be also (and in its current form).

Reply:
I agree with @plummercj , this is documenting that the flag is itself a "preview feature" and thus subject to change.

Reply:
I don't think it is correct to say that this command line option is a preview feature. Maybe it can be clarified on a second line, something like:

```
-trackvthreads    track virtual threads as they are created
                             Virtual threads are a preview API.
```

Reply:
Does that imply that `-trackvthreads` can be changed or removed?

Reply:
If virtual threads become permanent then the usage message would minimally be updated to drop the sentence that virtual threads are a preview feature. At that point the debugger APIs may have been built out further and it might be that there is no need for the -trackvthreads option but that's too far out to know at this point.

If virtual threads were to be removed then the -trackvthreads command line option would be removed too.

Reply:
> If virtual threads become permanent then the usage message would minimally be updated to drop the sentence that virtual threads are a preview feature.

But that's precisely the commitment to this flag that I want to avoid.  I want something that is the equivalent of `EXPERIMENTAL` for `-XX` flags.

Reply:
Again I have to say I agree with Chris. The flag itself is a "preview" flag and relates to a "preview feature". If the feature goes obviously the flag does too, but the flag could go even if the feature stays.

Reply:
Please see the discussion with Alex in [JDK-8286983](https://bugs.openjdk.java.net/browse/JDK-8286983)

Reply:
This PR has been updated to reflect the discussion in [JDK-8286983](https://bugs.openjdk.java.net/browse/JDK-8286983). Please re-review.